src/sdl/utils.cpp

```diff
@@ -726,33 +727,28 @@ bool mask_surface(surface& nsurf, const surface& nmask, const std::string& filen
 		return false;
 	}
 
-	bool empty = true;
+	uint8_t surf_alpha_sum{0};
```

Comment:
This isn't a sum, it's or'd bitwise. However, there's also the question of whether uint8_t is faster or slower than uint32_t - one of the tricks in my version was to use the same ARGB data as in the pixels themselves.